backup/run_backup.sh

```diff
@@ -45,13 +45,12 @@ tar -czvf $backup_file $backup_dir
 rm -rf $backup_dir
 
 # Backup to Dropbox
-$backup_to_dropbox $backup_file
-if [ $? -nq 0 ]; then
+if ! $backup_to_dropbox $backup_file; then
     echo DROPBOX BACKUP FAILURE!
     # Notify Slack
     if [[ -n "$SLACK_INCIDENTS_WH_URL" ]]; then
         curl -s -H 'content-type: application/json' \
-            -d "{ \"text\": \"❗DROPBOX BACKUP FAILURE❗\" }" \
+            -d "{ \"text\": \"❗DROPBOX BACKUP FAILURE❗\nBackup filename: $backup_file\" }" \
```

Comment:
It would be good to exit here since the `set -e` flag was removed. Would be good to not let the backup be deleted.